ray_on_golem/reputation/plugins/score.py

```diff
@@ -0,0 +1,88 @@
+import logging
+from typing import TYPE_CHECKING, Dict, Final, Optional, Sequence, Tuple
+
+from golem.managers.base import ProposalScorer, ProposalScoringResult
+from golem.utils.typing import MaybeAwaitable
+from tortoise.exceptions import DoesNotExist
+
+from ray_on_golem.reputation import models as m
+
+if TYPE_CHECKING:
+    from golem.resources import ProposalData
+
+logger = logging.getLogger(__name__)
+
+PARAM_UPTIME = "uptime"
+PARAM_SUCCESS_RATE = "success_rate"
+DEFAULT_UPTIME_WEIGHT: Final[float] = 1.0
+DEFAULT_SUCCESS_RATE_WEIGHT: Final[float] = 0.5
+
+
+class ReputationScorer(ProposalScorer):
+    def __init__(
+        self,
+        payment_network: str,
+        weights: Optional[Dict[str, float]] = None,
```

Comment:
Can you change this argument name to be more verbose. e.g. `param_weights`

Reply:
iow, to me it seems it's perfectly obvious what the weights are in the context of the reputation-based scorer...

Reply:
It is not obvious that this class is based on some parameters from external system. It was not clear at all what are the keys in this dictionary. `param_weights` correspond to `key: value` of this argument while `weights` only to `values`.

Reply:
gotcha, will update